src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -304,15 +147,12 @@ class BookingFacade(
   private fun Collection<PrisonAppointment>.main() = single { it.appointmentType == "VLB_COURT_MAIN" }
 
   private fun Collection<PrisonAppointment>.post() = singleOrNull { it.appointmentType == "VLB_COURT_POST" }
-
-  private fun PrisonAppointment.appointmentInformation(locations: Map<String, Location>) =
-    "${locations.room(prisonLocKey)} - ${startTime.toHourMinuteStyle()} to ${endTime.toHourMinuteStyle()}"
-
-  private fun Map<String, Location>.room(key: String) = this[key]?.localName ?: ""
 }
 
 enum class BookingAction {
```

Comment:
Need to think about where this lives as it is being used in the factory at the moment.